src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/BlockWrapOption.java

```diff
@@ -0,0 +1,66 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2017 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace;
+
+/**
+ * Represents the options for wrapping the contents of a code block.
+ *
+ * @author Pavel Bludov
+ * @see EmptyLineWrappingInBlockCheck
+ */
+public enum BlockWrapOption {
+
+    /**
+     * At the beginning/end of the block an empty string is allowed (no validation).
+     */
+    EMPTY_LINE_ALLOWED,
```

Comment:
By name alone, `EMPTY_LINE_ALLOWED` and `EMPTY_LINE`, mean the same thing.
I think this should be renamed to `NONE` to clearly state no validation will happen as the doc suggests.

> an empty string is allowed

I don't understand what an `empty string` means. We actually require an empty string (`""`)?

Reply:
done.